ui/package.json

```diff
@@ -141,6 +141,8 @@
   "dependencies": {
     "@influxdata/clockface": "0.0.13",
     "@influxdata/influx": "0.3.4",
+    "@influxdata/influxdb-template": "git+ssh://git@github.com:influxdata/influxdb-templates.git",
```

Comment:
should this line exist? Looks like it's being imported from the pluralized version instead (line 145)

Reply:
yeah missed that, thanks